app/assets/javascripts/components/messages_view.coffee

```diff
@@ -0,0 +1,17 @@
+@MessagesView = React.createClass
+  displayName: 'MessagesView'
+
+  render: ->
+    {ul, li, div} = React.DOM
+
+    return null unless @props.messages.length > 0
+
+    div
+      className: 'message-container'
+      ul
+        className: 'messages'
+        @props.messages.map (message, index) ->
+          li
+            className: 'message'
+            key: index
+            dangerouslySetInnerHTML: { __html: message }
```

Comment:
Er staat een link `mail naar support@roqua.nl` in. De reden dat de parameter een raw html string i.p.v een react-component is, is dat errors als React niet kon laden (bijv in IE8) direct door de browser getoond moeten worden.

In principe zou er als er een error tijdens initialisatie kwam ook wel een hardcoded bericht getoond kunnen worden, maar ik weet niet zo goed wat het in dit geval oplevert om geen `dangerouslySetInnerHTML` te gebruiken, de inhoud van de html is namelijk onafhankelijk van user input